chpl/chpl-service/src/main/resources/environment.properties.template

```diff
@@ -55,6 +55,9 @@ schemaCsv2014Name=listing2014Schema.csv
 schemaCsv2015Name=listing2015Schema.csv
 schemaSurveillanceName=surveillanceSchema.csv
 
+#all SED Details download folder file path
+downloadSEDFolderPath=C:\\Users\\blindsey\\downloadFiles\\SEDDetails\\chpl-sed-all-details.csv
```

Comment:
Why is this file not in the same directory as all of the other download files?

Reply:
Now that i think about it, i dont think i need a new environment variable at all...

Reply:
Right, there is already a downloadFolderPath property and you should just be providing a file name for the SED file expecting it to be found under downloadFolderPath.